Userland/Applications/Help/CMakeLists.txt

```diff
@@ -5,17 +5,14 @@ serenity_component(
     DEPENDS WebContent
 )
 
-stringify_gml(HelpWindow.gml HelpWindowGML.h help_window_gml)
+compile_gml(HelpWindow.gml HelpWindowGML.cpp)
 
 set(SOURCES
     History.cpp
     main.cpp
     MainWidget.cpp
     ManualModel.cpp
-)
-
-set(GENERATED_SOURCES
-    HelpWindowGML.h
+    HelpWindowGML.cpp
```

Comment:
Shouldn't this still be in `GENERATED_SOURCES`?

Reply:
No, generated sources is apparently only for generated header. The "copy file if different" handles this just fine.